geomstats/learning/mdm.py

```diff
@@ -0,0 +1,99 @@
+"""The MDM classifier on manifolds."""
+
+import geomstats.backend as gs
+from geomstats.learning.frechet_mean import FrechetMean
+
+
+class RiemannianMinimumDistanceToMeanClassifier():
+    """
+    Classifier implementing the MDM scheme on manifolds.
+
+    Attributes
+    ----------
+    riemannian_metric : string or callable.
+        The distance metric to use.
+    n_clusters: int, number of clusters.
+    mean_estimate: array-like, shape=[n_classes, n_features] if
+    point_type='vector'
+                   shape=[n_classes, n_features, n_features]
+                    if point_type='matrix'
+       Frechet means of each class.
+    """
+
+    def __init__(
+            self,
+            riemannian_metric,
+            n_clusters,
+            point_type='matrix'):
+        self.riemannian_metric = riemannian_metric
+        self.n_clusters = n_clusters
+        self.point_type = point_type
+        self.mean_estimate = None
+
+    def fit(self, X, Y):
```

Comment:
Y -> y to be consistent with scikit-learn notations

Reply:
Done.